jcommune-service/src/main/java/org/jtalks/jcommune/service/transactional/TransactionalPostService.java

```diff
@@ -149,11 +150,14 @@ public void deletePost(Post post) {
             branch.clearLastPost();
         }
 
-
         // todo: event API?
         topicDao.saveOrUpdate(topic);
         securityService.deleteFromAcl(post);
-        notificationService.subscribedEntityChanged(topic);
+
+        Set<JCUser> topicSubscribers = new HashSet<>(topic.getSubscribers());
+        topicSubscribers.remove(user);
```

Comment:
@vps2 the intention of the task is so that only the user who left that post needs to be notified about its removal. Your changes do the opposite - it'll notify everyone except the post creator :)